src/main/java/org/ecocean/shepherd/core/Shepherd.java

```diff
@@ -2875,141 +2876,6 @@ public int getNumMarkedIndividualsSightedAtLocationID(String locationID) {
         return getAllMarkedIndividualsSightedAtLocationID(locationID).size();
     }
 
```

Comment:
some of these methods like `getAllEncountersForSpecies()` were also removed in 1044. doesnt/shouldnt really matter at all; just making note of it in case something comes up with merging/conflicts later? i am just glad to get to see them go -- even twice!